docs/advanced_topics/custom_tasks.rst

```diff
@@ -195,6 +195,19 @@ For example:
             return TaskState.objects.none()
 
 
+``Task.get_description()```
+
+A class method that returns the human-readable description for the task.
+
+For example:
+
+.. code-block:: python
+
+    @classmethod
+    def get_description(cls):
+        return _("Choose which Wagtail Groups can approve this task")
```

Comment:
Might be nice to update this so it's a copy of the new wording?

Reply:
done